test/edu-benefits/1990-rjsf/config/educationHistory.unit.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester, getFormDOM } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1990-rjsf/config/form';
+
+describe('Edu 1990 educationHistory', () => {
+  const { schema, uiSchema } = formConfig.chapters.educationHistory.pages.educationHistory;
+  const definitions = formConfig.defaultDefinitions;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}
+          definitions={definitions}/>
+    );
+
+    const formDOM = getFormDOM(form);
+
+    expect(formDOM.querySelectorAll('input,select,textarea').length).to.equal(19);
+  });
+  it('should have no required inputs', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          onSubmit={onSubmit}
+          data={{}}
+          uiSchema={uiSchema}
+          definitions={definitions}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    formDOM.submitForm();
+
+    expect(Array.from(formDOM.querySelectorAll('.usa-input-error'))).to.be.empty;
```

Comment:
It looks like the native nodelist would work just fine here, and that the Array.from logic isn't necessary.